### ISSUE TITLE
Localhost:3000 is opening random thing.

### ISSUE THREAD

**Issue by aseem2625** [philholden/subdivide#19]
The project opens this page on ```npm start``` and opening ```localhost:3000/examples```
![image](https://cloud.githubusercontent.com/assets/6055628/21008625/84e1d6a0-bd68-11e6-83bd-e9234c66964a.png)

What's this?

**Comment by philholden** [comment]
Just an example that opens I frames in panes.

**Comment by aseem2625** [comment]
@philholden 
Nope. Nothing on this page looked like pane or frames! It looked like a someone's personal website to me. Please show a screenshot of it  where is the divider to sudivide!
Or correct if I am wrong, this library gives a divider for resizing!

**Comment by philholden** [comment]
Should appear when you hover over corners.
